features/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: test, ginkgo
+
+export FEATURES?=sctp 
+
+deps:
+	go mod tidy && \
+	go mod vendor
+
+ginkgo:
+	go get github.com/onsi/ginkgo/ginkgo
+
+functests: ginkgo
+	GOFLAGS=-mod=vendor ginkgo functests -- -junit /tmp/artifacts/unit_report.xml
+
+unittests:
+	go test $$(go list ./... | grep -v /functests)
+
+deploy:
+	for f in $(FEATURES) ; \
+    do \
+	  echo "deploying $$f"; \
+      ./$$f/deploy.sh; \
```

Comment:
@fedepaol nice :) I like this convention. IN the future this FEATUREs env var will change from executing deploy.sh to posting default CRs for the corresponding features.

Reply:
Nice direction indeed

Reply:
@fromanirh thanks! The idea is to leverage the same FEATURES variable to drive what e2e tests execute as well